packages/honeycomb-opentelemetry-web/src/types.ts

```diff
@@ -66,7 +66,7 @@ export interface HoneycombOptions extends Partial<WebSDKConfiguration> {
   endpoint?: string;
 
   /** Optionally pass extra headers to the exporter. Commonly used if sending to a collector that requires authentication */
-  headers?: { [key: string]: string | number };
+  headers?: { [key: string]: string };
```

Comment:
Technically this is a breaking change 😭 

The alternative is to map over `options.headers` in the `configureHeaders` function above and stringify any numeric values we get. 

That said, I would rather we stick to the otel API as closely as possible. 

Thoughts?

Reply:
I guess were still labeled as experimental but also we have this in the readme 

> Data shapes are stable and safe for production

If its not too hard then massaging the values as we are already doing a little seems ok.

But also I think it's fine to have this breaking change and call it out in the release notes. Maybe point fingers at the otel package that caused it as it looks like none of the packages increment a major version number for this

Reply:
Under semver, if you're still `0.x`, breaking changes can be included in a change in the minor version (ie. `0.84` can contain breaking changes from version `0.83`, which is exactly what happened [in this PR](https://github.com/open-telemetry/opentelemetry-js/pull/4971) and captured in their changelog [here](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/CHANGELOG.md#0540).

I'll merge this change as-is and link back to the OTel change. I'm also opening an issue against otel-js to update their changelog.

Reply:
https://github.com/open-telemetry/opentelemetry-js/issues/5125